### ISSUE TITLE
Feature Request: Add rsync

### ISSUE THREAD

**Issue by kartikynwa** [koreader/koreader#5668]
Device: Kobo Aura H2O

AFAIK Koreader doesn't come with rsync. I downloaded the binary from [this project](https://github.com/JBBgameich/rsync-static/releases) to `/usr/bin` and it works perfectly. With ssh available, rsync would be my preferred way to sync my library and maybe it would be helpful to other people as well so I thought I would suggest.

Thank you for reading this.

**Comment by NiLuJe** [comment]
I'd rather not end up maintaining the [same thing](https://www.mobileread.com/forums/showthread.php?t=254214) twice, but with *slightly* different platform-specific hacks like I had to do for dropbear, so, err, hard pass. ;).

There are *easy* way to get system-wide binaries of this stuff on probably all our supported platforms, so I'd strongly view that as out-of-scope, especially since that's one of those things you probably *don't* want a UI for.

What I have been toying with is a simple way to simply run .sh script from the FM, instead of having to rely on the Terminal plugin when you need to run a specific recurring task (like, say, an rsync sync ;p).

**Comment by Frenzie** [comment]
@m-majetic The armhf builds from Debian will normally Just Work™ on Kobo, which has traditionally been my approach rather than bothering to compile anything myself, but @NiLuJe's done one better and there's a large collection of Kobo optimized binaries here: https://www.mobileread.com/forums/showthread.php?t=254214 ("Yet another telnet/sshd & misc tools package...")

tl;dr: It sounds like you want rsync, so just use rsync. ;-)
